backend/src/main/java/quarano/sormas_integration/SormasClient.java

```diff
@@ -0,0 +1,104 @@
+package quarano.sormas_integration;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONException;
+import org.springframework.boot.context.properties.ConstructorBinding;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.stereotype.Component;
+import org.springframework.util.Base64Utils;
+import org.springframework.web.reactive.function.client.WebClient;
+import quarano.sormas_integration.indexcase.SormasCase;
+import quarano.sormas_integration.person.SormasContact;
+import quarano.sormas_integration.person.SormasPerson;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author Federico Grasso
+ */
+@Slf4j
+@EnableAsync
+@ConstructorBinding
+@RequiredArgsConstructor
+@Component
+public class SormasClient {
+
+    private WebClient client;
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private String sormasUrl;
+    private String sormasUser;
+    private String sormasPass;
+
+    public SormasClient(String sormasUrl, String sormasUser, String sormasPass) {
+
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        mapper.configure(MapperFeature.DEFAULT_VIEW_INCLUSION, true);
+
+        this.sormasUrl = sormasUrl;
+        this.sormasUser = sormasUser;
+        this.sormasPass = sormasPass;
+
+        client = WebClient
+                .create(this.sormasUrl);
+    }
+
+    private WebClient.ResponseSpec GetRequest(String route){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.